### PR TITLE
docs: add github documentation

### PR DESCRIPTION
**Pull request by shashidwivedi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
-# pillow-willow
-An e-commerce website for buying anything you will ever need in your home
+# Pillow Willow
+An e-commerce web app for buying anything you will ever need in your home
+
+[See Here](https://pillow-willow.netlify.app/)
+
+Screens:
+
+A homepage
```
Comment (ttahm3d) [inline README.md]:
```suggestion
Homepage
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
-# pillow-willow
-An e-commerce website for buying anything you will ever need in your home
+# Pillow Willow
+An e-commerce web app for buying anything you will ever need in your home
+
+[See Here](https://pillow-willow.netlify.app/)
```
Comment (ttahm3d) [inline README.md]:
```suggestion
Live Website: [https://pillow-willow.netlify.app/](https://pillow-willow.netlify.app/)]
```

**Review by ttahm3d — APPROVED** [review]
Suggested changes.

**Comment by shashidwivedi** [comment]
> Suggested changes.

Added the suggested changes

**Pull request merged**